test/fixtures/github/steps.yaml

```diff
@@ -12,6 +12,13 @@ jobs:
         with:
           repo: repository
 
+      - name: task with multiline params
+        uses: actions/checkout@v1
+        with:
+          repos: |
```

Comment:
You should add another input after this one to make sure that its line is calculated correctly